sdk-setup/src/mb2

```diff
@@ -2326,58 +2326,57 @@ run_deploy() {
     [[ ! $all_opt || $method_opt != --rsync ]] || fatal "Cannot use '--all' with '--rsync'"
     [[ ${method_opt:-} ]] || fatal "Argument expected"
 
-    local retcode=0
+    # rsync method is special
+    if [[ $method_opt == --rsync ]]; then
+        ! behind_qtcreator || run_make_install || return
+        local name=$(get_spec_tag "%{name}")
+        rsync_as $device_username -av ${buildroot}/. $device_ip:/opt/sdk/$name
```

Comment:
Shouldn't we check return value and print "Deploy failed" here if needed?

Reply:
I was thinking if the message adds any value at all. In the original code it is only printed if it fails at the last step, while it may fail on various errors before that. Drop it?

But what is more important, `dry_run_opt` should be handled here :D

Reply:
:-O Well I'm glad you reviewed your own code!